store/tikv/region_request.go

```diff
@@ -227,8 +225,6 @@ func (s *RegionRequestSender) SendReqCtx(
 	if span := opentracing.SpanFromContext(bo.GetCtx()); span != nil && span.Tracer() != nil {
 		span1 := span.Tracer().StartSpan("regionRequest.SendReqCtx", opentracing.ChildOf(span.Context()))
 		defer span1.Finish()
-		// TODO(MyonKeminta): Make sure trace works without cloning the backoffer.
-		// bo = bo.Clone()
 		bo.SetCtx(opentracing.ContextWithSpan(bo.GetCtx(), span1))
```

Comment:
Did you make sure the trace works well? I'm also not sure if it won't cause any problem that we  wraps too much layers on the bo.ctx. 🤔

Reply:
Yes.
```
MySQL [sbtest]> trace select * from sbtest1 where id = 1;
+-----------------------------------------------------------------------+-----------------+------------+
| operation                                                             | startTS         | duration   |
+-----------------------------------------------------------------------+-----------------+------------+
| trace                                                                 | 14:00:46.549722 | 7.401397ms |
|   ├─session.ExecuteStmt                                               | 14:00:46.549739 | 305.158µs  |
|   │ ├─executor.Compile                                                | 14:00:46.549754 | 34.52µs    |
|   │ └─session.runStmt                                                 | 14:00:46.549872 | 71.825µs   |
|   ├─*executor.PointGetExecutor.Next                                   | 14:00:46.550061 | 6.907209ms |
|   │ ├─tikvSnapshot.get                                                | 14:00:46.550068 | 6.884528ms |
|   │ └─regionRequest.SendReqCtx                                        | 14:00:46.550873 | 758.632µs  |
|   │   ├─rpcClient.SendRequest, region ID: 664100, type: Get           | 14:00:46.550905 | 662.145µs  |
|   │   └─tikv.onRegionError                                            | 14:00:46.551617 | 6.95µs     |
|   │     ├─tikv.backoff.regionMiss                                     | 14:00:46.551694 | 2.228813ms |
|   │     ├─loadRegion                                                  | 14:00:46.554161 | 542.379µs  |
|   │     └─regionRequest.SendReqCtx                                    | 14:00:46.554732 | 2.206014ms |
|   │       └─rpcClient.SendRequest, region ID: 671408, type: Get       | 14:00:46.554742 | 2.170395ms |
|   └─*executor.PointGetExecutor.Next                                   | 14:00:46.556995 | 2.212µs    |
+-----------------------------------------------------------------------+-----------------+------------+
14 rows in set (0.01 sec)
```